packages/client/src/peer-discovery/peer-discovery.ts

```diff
@@ -133,8 +141,7 @@ export class PeerDiscovery {
 
                 if (port >= 1 && port <= 65535) {
                     const peerData: IPeer = {
-                        ip: peer.ip,
-                        port,
+                        ...peer,
```

Comment:
In this case `peerData === peer` and code below for copying additional properties is useless, because all props are already copied. Or am I missing something?

```
if (opts.additional && Array.isArray(opts.additional)) {
  for (const additional of opts.additional) {
    if (typeof peer[additional] === "undefined") {
      continue;
    }

    peerData[additional] = peer[additional];
  }
}
```

Reply:
yeah, you are right. It's basically dead code.
Leftover from original codebase.
WIll remove it.